tests/notifier/notifier_test.py

```diff
@@ -28,6 +28,9 @@ class NotifierTest(ForsetiTestCase):
     def setUp(self):
         pass
 
+    def tearDown(self):
+        reload(notifier)
```

Comment:
What happens if you put the `reload(notifier)` in the `setUp()` and drop the reload() in line 146?

Because setUp() will run for every test case, before each test case executes?  Does it work?

If it does, that should be preferred.